Разметка всех страниц

eldarnugaev opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/298013/tasks/5)

Commit:
Размечает крупные смысловые блоки главной страницы

Commit:
Добавляет разметку шапки главной страницы

Commit:
Добавляет разметку основной части главной страницы

Commit:
Добавляет разметку футера главной страницы

Commit:
Добавляем страницу каталога с базовой разметкой внутри

Commit:
Добавляет разметку шапки и футера страницы каталога гостиниц

Commit:
Добавляет разметку основной части страницы каталога

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,255 @@
+          <ul class="hotel-list">
+            <li class="hotel-card">
+              <a class="hotel-card-link" href="#">
+                <img class="hotel-card-image" src="#" alt="">
```
alt нужно заполнять

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,255 @@
+                  <img src="#" alt="">
+                </li>
+                <li class="rating-item">
+                  <img src="#" alt="">
```
И тут везде alt пустой

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,255 @@
+                  <img src="#" alt="">
+                </li>
+                <li class="rating-item">
+                  <img src="#" alt="">
```
И тут. Пробегись по всем таким местам и посмотри, что добавить в alt

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,255 @@
+                <h3 class="hotel-card-title">Amara Resort & Spa</h3>
+                <dl>
+                  <dt>Гостиница</dt>
+                  <dd>От 4000 ₽</dd>
```
В подобных местах, желательно, тоже нужно вставлять неразрывный пробел (&nbsp;). Если ширина экрана будет нестандартной или просто маленькой, подобная конструкция может разъехаться

Commit:
Добавляет неразрывные пробелы в карточки отелей

Commit:
Добавляет alt к изображениям на странице каталога гостиниц

ghost commented:
@boumrz, внес исправления по заданию на разметку страниц.

boumrz commented on index.html:
```diff
@@ -5,8 +5,96 @@
+      </section>
+      <section class="advantage high-accent">
+        <h3>Настоящий<br>городок</h3>
+        <img class="advantage-devider" src="#" alt="">
```
Описание в alt не написал здесь и ниже

eldarnugaev replied:
Для декоративных картинок (разделители и иконки преимуществ) alt-текст решил не указывать по причине отсутствия смысла (в случае разделителей) и дублирования содержимого иконок соответствующим заголовком (в случае иконок преимуществ). Для всего остального (иконки в шапке и футере) alt-текст добавил.

boumrz replied:
Дело в том, что во всех картинках alt должен присутствовать

boumrz commented on index.html:
```diff
@@ -5,8 +5,96 @@
+      </section>
+    </main>
+    <footer class="page-footer">
+      <section class="subscribe">
```
Вот эта секция не в футере должна быть

eldarnugaev replied:
Вынес в область с основным содержимым main.

boumrz commented on index.html:
```diff
@@ -5,8 +5,96 @@
+      </section>
+      <section class="contacts">
+        <ul class="socials">
+          <li class="socials-item"><a class="socials-link" href="#"><span class="visually-hidden">Вконтакте</span></a></li>
```
В спан описание лишнее. И вообще, кажется, что тут не хватает img для картинок, так как эти изображения будут кликабельными

eldarnugaev replied:
Изменил разметку: вместо span сделал отдельными картинками.

Commit:
Изменяет разметку шапки и футера главной страницы

Добавляет в шапку иконки поиска и изранного.
Выносит блок с формой подписки из футера в основное содержимое main.
Добавляет иконки социальных сетей в футере.

Commit:
Изменяет разметку шапки и футера страницы каталога

Добавляет иконки поиска и избранного в шапку.
Выносит форму с подпиской из футера в основное содержимое main.
Добавляет в футер иконки социальных сетей.

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,255 @@
+      </nav>
+    </header>
+    <main class="main-catalog">
+      <header class="inner-header">
```
У тебя хедер уже есть на странице. Лучше не дублировать данный тег.

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,255 @@
+            <option value="expensive">Сначала дорогие</option>
+            <option value="popular">Сначала популярные</option>
+          </select>
+          <a class="sorting-grid" href="#"><span class="visually-hidden">Показать плиткой</span></a>
```
Ссылка нужна, чтобы была возможность перехода на другую страницу. Здесь скорее подойдет тег button

boumrz commented on index.html:
```diff
@@ -5,8 +5,96 @@
+      </section>
+      <section class="advantage low-accent">
+        <h3>Мало<br>туристов</h3>
+        <img src="" alt="">
```
Все alt в img должны быть заполненными в данном теге

boumrz reviewed (commented):
Хочется отметить хорошее решение - использовать неразрывный пробел в тех местах, где это необходимо. Ты делаешь все правильно в этом плане

Commit:
Исправляет замечания на странице каталога отелей

Убирает хедер внутри основного содержимого main.
Изменяет разметку переключения между отображениями результатов: реализация кнопками вместо ссылок.

Commit:
Исправляет замечания на главной странице

Добавляет alt-текст для декоративных изображений.
